config/recipes/elastic-agent/synthetic-monitoring.yaml

```diff
@@ -0,0 +1,159 @@
+---
+apiVersion: agent.k8s.elastic.co/v1alpha1
+kind: Agent
+metadata:
+  name: fleet-server
+  namespace: default
+spec:
+  version: 8.16.0
+  kibanaRef:
+    name: kibana
+  elasticsearchRefs:
+    - name: elasticsearch
+  mode: fleet
+  fleetServerEnabled: true
+  policyID: eck-fleet-server
+  deployment:
+    replicas: 1
+    podTemplate:
+      spec:
+        serviceAccountName: elastic-agent
+        automountServiceAccountToken: true
+        securityContext:
+          runAsUser: 0
+        volumes:
+        - name: agent-data
+          emptyDir: {}
+---
+apiVersion: agent.k8s.elastic.co/v1alpha1
+kind: Agent
+metadata:
+  name: synthetics-agent
+  namespace: default
+spec:
+  version: 8.16.0
+  image: docker.elastic.co/elastic-agent/elastic-agent-complete:8.16.0
+  kibanaRef:
+    name: kibana
+  fleetServerRef:
+    name: fleet-server
+  mode: fleet
+  policyID: synthetics-server
+  deployment:
+    replicas: 1
+    podTemplate:
+      spec:
+        serviceAccountName: elastic-agent
+        automountServiceAccountToken: true
+        securityContext:
+          runAsUser: 0
+        volumes:
+        - name: agent-data
+          emptyDir: {}
+---
+apiVersion: kibana.k8s.elastic.co/v1
+kind: Kibana
+metadata:
+  name: kibana
+  namespace: default
+spec:
+  version: 8.16.0
+  count: 1
+  elasticsearchRef:
+    name: elasticsearch
+  config:
+    xpack.fleet.agents.elasticsearch.hosts:
+    - "https://elasticsearch-es-http.default.svc:9200"
+    xpack.fleet.agents.fleet_server.hosts:
+    - "https://fleet-server-agent-http.default.svc:8220"
+    xpack.fleet.packages:
+      - name: elastic_agent
+        version: latest
+      - name: fleet_server
+        version: latest
+      - name: synthetics
+        version: latest
+    xpack.fleet.agentPolicies:
+      - name: Synthetics Server
+        id: synthetics-server
+        namespace: default
+
```

Comment:
```suggestion
        is_managed: true
        unenroll_timeout: 900
        package_policies: []
```

` package_policies: []` is to avoid the following error in Kibana when displaying the Agents:

<img width="1653" alt="image" src="https://github.com/user-attachments/assets/22ed41f6-cbde-45b8-b004-b2492ef51667" />